sdk/mixedreality/mixedreality-authentication/package.json

```diff
@@ -0,0 +1,119 @@
+  "description": "Azure client library for Mixed Reality authentication.",
+  "sdk-type": "client",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "browser": {
+    "./dist-esm/test/utils/env.js": "./dist-esm/test/utils/env.browser.js"
+  },
+  "types": "types/mixedreality-authentication.d.ts",
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
+    "build:samples": "dev-tool samples prep && cd dist-samples && tsc -p .",
+    "build:test": "tsc -p . && rollup -c 2>&1",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
+    "check-format": "prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-* test-dist temp types *.tgz *.log",
+    "execute:samples": "npm run build:samples && dev-tool samples run dist-samples/javascript dist-samples/typescript/dist/dist-samples/typescript/src/",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "generate:client": "autorest --typescript --v3 swagger/README.md",
+    "integration-test:browser": "karma start --single-run",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}/*.spec.js\"",
+    "integration-test": "npm run integration-test:node && npm run integration-test:browser",
+    "lint:fix": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts --fix --fix-type [problem,suggestion]",
+    "lint": "eslint package.json tsconfig.json api-extractor.json src test --ext .ts -f html -o mixedreality-authentication-lintReport.html || exit 0",
+    "pack": "npm pack 2>&1",
+    "prebuild": "npm run clean",
+    "test:browser": "npm run build:test && npm run unit-test:browser && npm run integration-test:browser",
+    "test:node": "npm run build:test && npm run unit-test:node && npm run integration-test:node",
+    "test": "npm run build:test && npm run unit-test && npm run integration-test",
+    "unit-test:browser": "karma start --single-run",
+    "unit-test:node": "mocha -r esm --require ts-node/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 1200000 --full-trace \"test/{,!(browser)/**/}/*.spec.ts\"",
+    "unit-test": "npm run unit-test:node && npm run unit-test:browser",
+    "docs": "typedoc --excludePrivate --excludeNotExported --excludeExternals --mode file --out ./dist/docs ./src"
```

Comment:
You might want to run this locally and take a look at the docs that are generated. Alternatively you can look at your js - mixedreality - ci build and download the documentation artifacts to see what the pipeline generates. There's a lot more showing up there than what you're exporting from `index.ts`.

TypeDoc unfortunately will look at exports and top-level symbols from all files, not just the entry point, so anything you don't want documented should be annotated with a `@internal` doc tag.

Reply:
You're absolutely right. There's a ton showing up that I didn't intend to. I can add tags to eliminate some things, but the generated code I can't fix. Any suggestions for excluding the generated content?

Reply:
Should I be excluding `SDK_VERSION` as well?

Reply:
As you mentioned offline, i've added `@internal` to the `SDK_VERSION` as well.

Reply:
I talked to @deyaaeldeen about the generated content. He said that the docs that end up on docs.microsoft.com may have a different process that we can inquire about since they may be removing those exports themselves.

@deyaaeldeen Do you happen to have a contact?

I don't think we should block merging this based on generated docs showing up in the TypeDoc output. @craigktreasure can you create an issue just to track following up with the docs team on how to prevent generated content from showing up? (Unless you want to do that before merging.)

Reply:
@chradek This is how the docs team generate docs: https://apidrop.visualstudio.com/Content%20CI/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=3452 but I did not have the time to look through all the steps and their code yet.

Reply:
And this is the package they mainly use for generating latest docs: https://apidrop.visualstudio.com/Content%20CI/_git/ReferenceAutomation?path=%2Fnodejs%2FJavaScriptTypeScript%2Fsrc%2Fmain.ts

Reply:
@chradek I've filed #13765. I don't see a need to block on it since all the other libraries will have the same issue. Happy to address it if there is a need and when there is a solution.